content/references/rippled-api/validator-list.md

```diff
@@ -0,0 +1,124 @@
+# Validator List Method
+
+The validator list method is a special API endpoint for fetching the latest version of a current trusted validator list a `rippled` server is using. This often represents the exact list of validators a server trusts. [New in: rippled 1.5.0][]
```

Comment:
Is it grabbing the latest version (as in number)? Or that latest VL itself? 

If it's the former, consider this tweak: 

> The validator list method is a special API endpoint that fetches the latest version of a trusted validator list in use by a `rippled` server.

If it's the latter: 

> The validator list method is a special API endpoint that fetches the latest trusted validator list in use by a `rippled` server.

Reply:
Since a server can use more than one list, I don't want to say "the latest... list" but I see your point. Going with:

>... that fetches a current, trusted validator list ...